client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -565,9 +569,10 @@ class JetpackThankYouCard extends Component {
 						className={ classNames( 'button', 'thank-you-card__button', {
 							'is-placeholder': ! buttonUrl,
 						} ) }
+						onclick={ this.onBackToYourSiteClick }
 						href={ buttonUrl }
 					>
-						{ translate( 'Explore your plan' ) }
+						{ translate( 'Back to your site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Back to my site' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/jetpack-back-text)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).